crates/release_channel/src/lib.rs

```diff
@@ -151,9 +151,9 @@ impl ReleaseChannel {
     /// This also has to match the bundle identifier for Zed on macOS.
     pub fn app_id(&self) -> &'static str {
         match self {
-            ReleaseChannel::Dev => "dev.zed.Zed-Dev",
-            ReleaseChannel::Nightly => "dev.zed.Zed-Nightly",
-            ReleaseChannel::Preview => "dev.zed.Zed-Preview",
+            ReleaseChannel::Dev => "dev.zed.ZedDev",
```

Comment:
There are a few more spots which you should update the IDs in:

- Line 100 of `crates/collab/src/api/events.rs`
- Line 68 and onward of `scripts/install.sh`
- Line 15 and onward of `script/flatpak/bundle-flatpak`

Reply:
I think I got all of them now.